docs/API_SPECS_DEVICE_MODELS_V2.md

```diff
@@ -0,0 +1,488 @@
+API specifications - Device Models v2
```

Comment:
should we merge this file into the other md?

Reply:
Done.